crypto/perlasm/x86asm.pl

```diff
@@ -286,7 +286,7 @@ sub ::asm_init
     {	print STDERR <<"EOF";
 Pick one target type from
 	elf	- Linux, FreeBSD, Solaris x86, etc.
-	a.out	- DJGPP, elder OpenBSD, etc.
+	a.out	- elder OpenBSD, etc.
```

Comment:
I only know of a single person who might still care about some pre-ELF OpenBSD, but that person won't care about x86. :)

Reply:
Should I change it to s/elder OpenBSD/You know who you are/ ? :)
More seriously, remove a.out ?

Reply:
I'd just ditch the line.